src/status_im/chat/commands/impl/transactions.cljs

```diff
@@ -369,67 +378,55 @@
-          on-press-handler  (cond
-                              network-mismatch?
-                              nil
-                              (and (not answered?)
-                                   status-initialized?)
-                              #(re-frame/dispatch [:select-chat-input-command
-                                                   command
-                                                   [(or asset "ETH") amount]
-                                                   {:responding-to message-id}]))]
-      [react/view
-       [react/touchable-highlight
-        {:on-press on-press-handler}
-        [react/view (transactions-styles/command-request-message-view outgoing)
-         [react/view
-          [react/view
-           [react/text {:style (transactions-styles/command-request-header-text outgoing)}
-            (i18n/label :transaction-request)]]
-          [react/view transactions-styles/command-request-row
-           [react/text {:style transactions-styles/command-request-amount-text
-                        :font  :medium}
-            amount
-            [react/text {:style (transactions-styles/command-amount-currency-separator outgoing)}
-             "."]
-            [react/text {:style (transactions-styles/command-request-currency-text outgoing)
-                         :font  :default}
-             asset]]]
-          [react/view transactions-styles/command-request-fiat-amount-row
-           [react/text {:style transactions-styles/command-request-fiat-amount-text}
-            (str "~ " fiat-amount " " (or currency (i18n/label :usd-currency)))]]
-          (when (and group-chat recipient-name)
-            [react/text {:style transactions-styles/command-request-recipient-text}
-             (str
-              (i18n/label :request-requesting-from)
-              " "
-              recipient-name)])
-          (when network-mismatch?
-            [react/text {:style transactions-styles/command-request-network-text}
-             (str (i18n/label :on) " " request-network)])
-          [react/view transactions-styles/command-request-timestamp-row
-           [react/text {:style (transactions-styles/command-request-timestamp-text outgoing)}
-            (str
-             (datetime/timestamp->mini-date timestamp)
-             " "
-             (i18n/label :at)
-             " "
-             timestamp-str)]]
-          (when-not outgoing
-            [react/view
-             [react/view transactions-styles/command-request-separator-line]
-             [react/view transactions-styles/command-request-button
-              [react/text {:style    (transactions-styles/command-request-button-text answered?)
-                           :on-press on-press-handler}
-               (i18n/label (if answered? :command-button-sent :command-button-send))]]])]]]])))
+          markup            [react/view (transactions-styles/command-request-message-view outgoing)
```

Comment:
couldn't it be defined as a view in screens or component, taking params as input ?